app/views/branded/api/v3/orgs/_show.json.jbuilder

```diff
@@ -29,7 +29,7 @@ if org.is_a?(RegistryOrg)
   else
     json.affiliation_id do
       json.type 'ror'
-      json.identifier "https://ror.org/#{org.ror_id}"
+      json.identifier org.ror_id.include?('ror.org') ? org.ror_id : "https://ror.org/#{org.ror_id}"
```

Comment:
@briri could there be any other permutations of` ror_id` other than `https://ror.org/1234567` or `1234567`?

Reply:
theoretically I guess there could be `ror.org/1234567`. I'll make a change to just strip out the base domain and protocol and always prepend it.